src/rebar_templater.erl

```diff
@@ -162,7 +162,14 @@ create({Template, Type, File}, Files, UserVars, Force, State) ->
     Vars = drop_var_docs(override_vars(UserVars, get_template_vars(TemplateTerms, State))),
     maybe_warn_about_name(Vars),
     TemplateCwd = filename:dirname(File),
-    execute_template(TemplateTerms, Files, {Template, Type, TemplateCwd}, Vars, Force).
+    Result = execute_template(TemplateTerms, Files, {Template, Type, TemplateCwd}, Vars, Force),
+    maybe_print_final_message(proplists:get_value(message, TemplateTerms, undefined), Vars),
+    Result.
+
+maybe_print_final_message(undefined, _) ->
+    ok;
+maybe_print_final_message(Message, Values) ->
+    io:format("Usage:~n~n~s~n", [render(Message, Values)]).
```

Comment:
Do we want to have the `Usage:` prefix always present?

Reply:
Good point, I guess the message should be able to be for any purpose.